QuestionsforKaida_s2.md

```diff
@@ -0,0 +1,6 @@
+- How can I refactor the code without repeating the  footer and nav for each page?
+- What is the best way to import google fonts? link or @url import 
+- How to target the right element? When should I target the parent and when should I target the child? sometimes it seems interchangeable
+- Which elements should be "stored" in containers?
```

Comment:
My answer personally is the ones that need it.  

I try to avoid extra containers that aren't necessary for breaking the document into logical chunks (or visual chunks).